### ISSUE TITLE
Needs setup.py to be able to install over Pip

### ISSUE THREAD

**Issue by rfkrocktk** [dcramer/mangodb#45]
Critical infrastructure depends on web-scale software, how can I automate installation if I can't do:

    sudo pip install git+https://github.com/wcramer/mangodb.git

**Comment by judy2k** [comment]
Can we make it work with easy_install? I fear change.

**Comment by rfkrocktk** [comment]
As long as I can run a single command so I can automate my mistakes.

Thanks,
 - Naftuli Tzvi

**Comment by judy2k** [comment]
Agreed. Automation is very important in enterprise web scale software.

**Comment by d0ugal** [comment]
How do I do this with docker?
